### PR TITLE
multimc: update url, add depends_on

### PR DESCRIPTION
**Pull request by bevanjkay**
Continuation from #111250 111250

### CODE REVIEW

**Comment by bevanjkay** [comment]
Ping @haykam821

**Review by haykam821 — APPROVED** [review]

**Comment by haykam821** [comment]
Thank you! This should fix issues with dark mode and using Microsoft accounts.

File: Casks/multimc.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   version "0.6.13"
   sha256 :no_check
 
-  url "https://files.multimc.org/downloads/mmc-stable-osx64.tar.gz"
+  url "https://files.multimc.org/downloads/mmc-stable-osx64-5.5.12.tar.gz"
```
Comment (haykam821) [inline Casks/multimc.rb:5]:
```suggestion
  url "https://files.multimc.org/downloads/mmc-stable-osx64-5.15.2.tar.gz"
```

**Review by haykam821 — CHANGES_REQUESTED** [review]

**Comment by bevanjkay** [comment]
@haykam821 I will wait for other maintainer feedback before merging as this version is not the version provided by the `downloads` page on the upstream website. We will need to make the livecheck match with the version that is being downloaded also.

**Pull request merged**